zombies.js

```diff
@@ -49,13 +60,18 @@
  * @property {number} energy
  */
 
-
 /**
  * Food Extends Item Class
  * -----------------------------
  */
 
-
+class Food extends Item {
+  constructor(name, energy) {
+    super(name);
+    this.name = name;
```

Comment:
This extends Item so it doesn't need to set the name. When you call the super, it's running Item's constructor which is going to set the name for you. That's why we pass the name into super.